scripts/exhafs_enkf_hx.sh

```diff
@@ -438,9 +440,28 @@ ${NLN} ${COMINhafs_obs}/hafs.t${cyc}z.tldplr.tm00.bufr_d          tldplrbufr
 
 fi #USE_SELECT
 
+# Workflow will read from previous cycles for satbias predictors if online_satbias is set to yes
+if [ ${online_satbias} = "no" ] && [ ${RUN_ENVAR} = "YES" ]; then
```

Comment:
@jak5808 I am still somewhat confused. Should this if condition be the following?
`if [ ${online_satbias} = "yes" ] && [ ${RUN_ENVAR} = "YES" ]; then`